packages/moleculer-bull/src/index.js

```diff
@@ -51,15 +50,94 @@ module.exports = function createService(url, queueOpts) {
 					this.$queues[name] = Queue(name, url, queueOpts);
 				}
 				return this.$queues[name];
+			},
+
+			/**
+			 * Set metrics for every queue
+			 * @param name
+			 */
+			addMetrics(name) {
+				// Collect metrics
+				try {
+					const queue = this.getQueue(name);
+					setInterval(async () => {
+						const {completed, active, delayed, failed, waiting} = await queue.getJobCounts();
+						this.broker.metrics.set("bull.completed", completed);
+						this.broker.metrics.set("bull.active", active);
+						this.broker.metrics.set("bull.delayed", delayed);
+						this.broker.metrics.set("bull.failed", failed);
+						this.broker.metrics.set("bull.waiting", waiting);
+					}, 1000);
```

Comment:
Somebody should stop the intervals.
**Option 1:** collect the timers in a service variables and stopping in the `stopped` lifecycle hook.
**Option 2:** Use `.unref()` _(not the best in performance)_
**Option 3:** Use only one timer what you start in `started` and stop in `stopped` and in ticks forEach-ing the `this.$queues` and set the queue values. _(best option)_